moveit_ros/planning/trajectory_execution_manager/test/test_execution_manager.cpp

```diff
@@ -60,15 +60,21 @@ class MoveItCppTest : public ::testing::Test
     trajectory_execution_manager_ptr = moveit_cpp_ptr->getTrajectoryExecutionManagerNonConst();
 
     traj1.joint_trajectory.joint_names.push_back("panda_joint1");
-    traj1.joint_trajectory.points.resize(1);
+    traj1.joint_trajectory.points.resize(2);
     traj1.joint_trajectory.points[0].positions.push_back(0.0);
+    traj1.joint_trajectory.points[1].positions.push_back(0.5);
+    traj1.joint_trajectory.points[1].time_from_start.fromSec(0.5);
 
     traj2 = traj1;
     traj2.joint_trajectory.joint_names.push_back("panda_joint2");
     traj2.joint_trajectory.points[0].positions.push_back(1.0);
+    traj2.joint_trajectory.points[1].positions.push_back(1.5);
     traj2.multi_dof_joint_trajectory.joint_names.push_back("panda_joint3");
-    traj2.multi_dof_joint_trajectory.points.resize(1);
+    traj2.multi_dof_joint_trajectory.points.resize(2);
     traj2.multi_dof_joint_trajectory.points[0].transforms.resize(1);
+    traj2.multi_dof_joint_trajectory.points[1].transforms.resize(1);
+
+    ros::param::del("~/trajectory_execution/joints_allowed_start_tolerance");
```

Comment:
I want to be sure the tests don't start with a value from another test. For example if `RejectTooFarFromStart` were to run after `DoNotValidateJointStartToleranceZero` without reseting this value, it would fail.

Reply:
`setAllowedStartTolerance` doesn't change `joints_allowed_start_tolerance`

Reply:
OK. I didn't notice, that you delete the map.